superduperdb/backends/local/compute.py

```diff
@@ -0,0 +1,59 @@
+import typing as t
+
+from superduperdb import logging
+from superduperdb.backends.base.compute import ComputeBackend
+
+
+class LocalComputeBackend(ComputeBackend):
+    """
+    A mockup backend for running jobs locally.
+    """
+
+    def __init__(
+        self,
+    ):
+        self.futures_collection: t.Dict[str, t.Any] = {}
+
+    def name(self) -> str:
+        return "Local"
+
+    def submit(self, function: t.Callable, **kwargs) -> None:
+        """
+        Submits a function for local execution.
+
+        :param function: The function to be executed.
+        :param kwargs: Additional keyword arguments to be passed to the function.
+        """
+        future = function(**kwargs)
```

Comment:
lets rename this to output
and in __init__ 
self.outputs = {}

self.outputs[function.__name__] = future